src/main/kotlin/no/nav/tilleggsstonader/sak/journalføring/dto/JournalføringRequest.kt

```diff
@@ -0,0 +1,74 @@
+package no.nav.tilleggsstonader.sak.journalføring.dto
+
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.kontrakter.journalpost.LogiskVedlegg
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingÅrsak
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeilHvis
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.feilHvis
+import java.time.LocalDate
+
+data class JournalføringRequest(
+    val dokumentTitler: Map<String, String>? = null,
+    val logiskeVedlegg: Map<String, List<LogiskVedlegg>>? = null,
```

Comment:
Håndterer dere logiske vedlegg nå? Eller kommer det senere?

Reply:
Det er lagt til støtte for det fra frontend sin side i hvertfall, tror det kan legges inn